src/web/app/components/question-types/question-edit-details-form/text-question-edit-details-form.component.ts

```diff
@@ -14,10 +14,21 @@ import { QuestionEditDetailsFormComponent } from './question-edit-details-form.c
 export class TextQuestionEditDetailsFormComponent extends QuestionEditDetailsFormComponent<FeedbackTextQuestionDetails>
     implements OnInit {
 
+  shouldAllowRichText: boolean = true;
```

Comment:
You can remove the state as it is tracked with `model`

Reply:
This is used per question. I am not sure if I could access the ngModel here
`this.shouldAllowRichText = richTextAllowed;` at line 31?